src/CppUTest/SimpleString.cpp

```diff
@@ -631,7 +631,7 @@ SimpleString StringFromBinaryWithSizeOrNull(const unsigned char* value, size_t s
 SimpleString StringFromMaskedBits(unsigned long value, unsigned long mask, size_t byteCount)
 {
     SimpleString result;
-    size_t bitCount = (byteCount > sizeof(unsigned long)) ? (sizeof(unsigned long) * 8) : (byteCount * 8);
+    size_t bitCount = (byteCount > ROBUST_SIZEOF(unsigned long)) ? (ROBUST_SIZEOF(unsigned long) * 8) : (byteCount * 8);
     const unsigned long msbMask = (((unsigned long) 1) << (bitCount - 1));
 
     for (size_t i = 0; i < bitCount; i++) {
```

Comment:
If we get rid of the macro, we need to get rid of the magical number "8" as well. Perhaps define a macro like BITS_PER_BYTE which will be 8 for most systems and 16 for systems with 16 bit bytes.

How and where do you suggest this be defined?